release/models/local-routing/openconfig-local-routing.yang

```diff
@@ -221,7 +227,7 @@ module openconfig-local-routing {
 
     leaf next-hop {
       type union {
-        type inet:ip-address;
+        type inet:ip-address-zoned;
```

Comment:
The way the regex validation is written for ip-address-zoned, it requires a zone indicator.  So this really needs to be a union of ip-address and ip-address-zoned

Reply:
@nokia1adam, I think this is the only change needed and it will be ready.  Please update to use a union of both the current ip address format and the zoned format.

Reply:
@nokia1adam would you like to update this?